README.md

```diff
@@ -74,23 +120,55 @@ if __name__ == "__main__":
         print("error occurred.", exc)
 ```
 
-#### Run File Uploader
+**1. Create a test file containing data:**
+
+You can do this with `dd` on Linux or macOS systems:
+
 ```sh
-$ python file_uploader.py
-'/home/user/Photos/asiaphotos.zip' is successfully uploaded as object 'asiaphotos-2015.zip' to bucket 'asiatrip'.
+dd if=/dev/urandom of=/tmp/minio-python-sdk-test-file.bin bs=2048 count=10
+```
+
+or `fsutil` on Windows:
 
-$ mc ls play/asiatrip/
-[2016-06-02 18:10:29 PDT]  82KiB asiaphotos-2015.zip
+```sh
+fsutil file createnew "C:\Users\<username>\Desktop\minio-python-sdk-test-file.bin" 20480
+```
+
+**2. Run `file_uploader.py` with the following command:**
+
+```sh
+python file_uploader.py
+```
+
+The output resembles the following:
+
+```sh
+/tmp/minio-python-sdk-test-file.bin successfully uploaded as object my-test-file.bin to bucket minio-python-sdk-test-bucket
+```
+
+**3. Verify the Uploaded File With `mc ls`:**
+
+```sh
+mc ls play/minio-python-sdk-test-bucket
+[2023-11-03 22:18:54 UTC]  20KiB STANDARD my-test-file.bin
 ```
 
 ## More References
+
 * [Python Client API Reference](https://min.io/docs/minio/linux/developers/python/API.html)
 * [Examples](https://github.com/minio/minio-py/tree/master/examples)
 
 ## Explore Further
+
 * [Complete Documentation](https://min.io/docs/minio/kubernetes/upstream/index.html)
 
 ## Contribute
-Please refer [Contributors Guide](https://github.com/minio/minio-py/blob/master/CONTRIBUTING.md)
+
+[Contributors Guide](https://github.com/minio/minio-py/blob/master/CONTRIBUTING.md)
+
+## License
+
+This SDK is distributed under the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0), see [LICENSE](https://github.com/minio/minio-py/blob/master/LICENSE) for more information.
```

Comment:
The Go one also links to its `NOTICE` but this repo doesn't have that file. Recommendations?

Reply:
You could copy the file from `minio-go` to here and have a link.